gnupg.pyi

```diff
@@ -0,0 +1,890 @@
+"""
+This type stub file was generated by pyright.
+"""
+
+from typing import Any
+
+__version__: str = ...
+__author__: str = ...
+__date__: str = ...
+
```

Comment:
I would add something like this:

```suggestion

class ReadableFile(Protocol):
    def read(self, size: int) -> bytes: ...
    
```

and then use it to annotate all the `fileobj_or_path` parameters as `str | ReadableFile`.